index.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
-var isFinite = require('is-finite');
+module.exports = (n, str) => {
+	str = str || ' ';
```

Comment:
I can't think of any use-case, but users might do an empty string `''` and expect it to not indent. Better to be safe here and rather check if it's undefined.